shell/browser/electron_browser_client.cc

```diff
@@ -1350,6 +1376,19 @@ bool ElectronBrowserClient::WillCreateURLLoaderFactory(
     bool* bypass_redirect_checks,
     bool* disable_secure_dns,
     network::mojom::URLLoaderFactoryOverridePtr* factory_override) {
+#if BUILDFLAG(ENABLE_ELECTRON_EXTENSIONS)
+  auto* web_request_api =
+      extensions::BrowserContextKeyedAPIFactory<extensions::WebRequestAPI>::Get(
+          browser_context);
+
+  // NOTE: Some unit test environments do not initialize
+  // BrowserContextKeyedAPI factories for e.g. WebRequest.
+  if (web_request_api) {
+    web_request_api->MaybeProxyURLLoaderFactory(
+        browser_context, frame_host, render_process_id, type,
+        std::move(navigation_id), factory_receiver, header_client);
```

Comment:
It seems like you'd need to `return` here, otherwise we'd try to also pass the request through Electron's `webRequest` API, which I don't think would work. Not least, it would result in double-`std::move`ing of `navigation_id`.

Reply:
Well, I was thinking of keeping support for both Electron's `webRequest` and `chrome.webRequest`.

Reply:
I don't know, but the code as it is does not do that correctly :)

Reply:
I assume that this would be tricky, so what I propose for now is that registering any `chrome.webRequest` would override the Electron's `webRequest` module. But I think that's not the behavior we want?